scripts/west_commands/runners/pyocd.py

```diff
@@ -111,15 +111,20 @@ def do_run(self, command, **kwargs):
         else:
             self.debug_debugserver(command, **kwargs)
 
-    def flash(self, **kwargs):
+    def _getfname(self):
         if os.path.isfile(self.hex_name):
-            fname = self.hex_name
-        elif os.path.isfile(self.bin_name):
-            fname = self.bin_name
-        else:
-            raise ValueError(
-                'Cannot flash; no hex ({}) or bin ({}) files'.format(
-                    self.hex_name, self.bin_name))
+            return self.hex_name
+
+        if os.path.isfile(self.bin_name):
+            self.logger.warning('Hex file "{}" not found, falling back to binary "{}"'.format(self.hex_name, self.bin_name))
```

Comment:
Please keep this file 80-column clean.

Reply:
Why was this comment marked resolved? Did you forget to push an update?

Reply:
I have no idea what I'm doing. Never used this feature.

Reply:
Thought resolving would apply the suggestion